xbmc/interfaces/legacy/InfoTagVideo.h

```diff
@@ -39,108 +67,495 @@ namespace XBMCAddon
       InfoTagVideo();
       virtual ~InfoTagVideo();
 
+#ifndef SWIG /* Part below used for HTML doc and behind #else for python internal docs */
+      ///
+      /// \ingroup python_InfoTagVideo
+      /// @brief Get identification number of tag in database
+      ///
+      /// @return [integer] database id
+      ///
+      ///
+      ///------------------------------------------------------------------------
+      ///
+      ///
+      int getDbId();
+#else
       /**
        * getDbId() - returns an integer.
        */
       int getDbId();
+#endif
+
+#ifndef SWIG /* Part below used for HTML doc and behind #else for python internal docs */
+      ///
+      /// \ingroup python_InfoTagVideo
+      /// @brief Get [film director](https://en.wikipedia.org/wiki/Film_director)
+      /// who has made the film (if present).
+      ///
+      /// @return [string] Film director name.
+      ///
+      ///
+      ///------------------------------------------------------------------------
+      ///
+      ///
+      String getDirector();
+#else
       /**
        * getDirector() - returns a string.
        */
       String getDirector();
+#endif
+
+#ifndef SWIG /* Part below used for HTML doc and behind #else for python internal docs */
+      ///
+      /// \ingroup python_InfoTagVideo
+      /// @brief Get the writing credits if present from video info tag.
+      ///
+      /// @return [string] Writing credits
+      ///
+      ///
+      ///------------------------------------------------------------------------
+      ///
+      ///
+      String getWritingCredits();
+#else
       /**
        * getWritingCredits() - returns a string.
        */
       String getWritingCredits();
+#endif
+
+#ifndef SWIG /* Part below used for HTML doc and behind #else for python internal docs */
+      ///
+      /// \ingroup python_InfoTagVideo
+      /// @brief To get the [Video Genre](https://en.wikipedia.org/wiki/Film_genre)
+      /// if available.
+      ///
+      /// @return [string] Genre name
+      ///
+      ///
+      ///------------------------------------------------------------------------
+      ///
+      ///
+      String getGenre();
+#else
       /**
        * getGenre() - returns a string.
        */
       String getGenre();
+#endif
+
+#ifndef SWIG /* Part below used for HTML doc and behind #else for python internal docs */
+      ///
+      /// \ingroup python_InfoTagVideo
+      /// @brief Get video tag line if available.
+      ///
+      /// @return [string] Video tag line
+      ///
+      ///
+      ///------------------------------------------------------------------------
+      ///
+      ///
+      String getTagLine();
+#else
       /**
        * getTagLine() - returns a string.
```

Comment:
Python documentation seems to be very unclair to me.
@AlwinEsch  wrote 
```
/// @brief Get video tag line if available.
///
/// @return [string] Video tag line
```
Current Documentation says:
```
* getTagLine() - returns a string.
```